integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -45,6 +45,9 @@ <h1 class="heading">
                 {% endif %}
             </h1>
             <div class="flex flex-wrap grow justify-end gap-2 items-center">
+                {% if page_form.instance.id and not page_form.instance.archived and can_edit_page %}
```

Comment:
```suggestion
                {% if not page_form.instance.id and not page_form.instance.archived and can_edit_page %}
```
On the old location it was inside a if statement with `if not page_form.instance.id`. Is there a specific reason to change that?

Reply:
This was a requested change by Mizuki to be consistent in all three content forms